### PR TITLE
CA refactor - move logic out of views

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

File: web-client/src/ustc-ui/utils/documentTypeSelectHelper.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+export const fileDocumentSecondaryOnChange = ({
+  action,
+  inputValue,
+  updateSequence,
+  validateSequence,
+}) => {
+  switch (action) {
+    case 'select-option':
+      updateSequence({
```
Comment (kkoskelin) [inline web-client/src/ustc-ui/utils/documentTypeSelectHelper.js]: I'd be tempted to reduce it further.
```
['category','documentType',...].forEach(key => 
  updateSequence({
    key: `secondaryDocument.${key}`,
    value: inputValue[key]
  })
);
```

File: web-client/src/ustc-ui/utils/documentTypeSelectHelper.test.js [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+        value: inputValue.eventCode,
+      });
+      expect(updateSequenceSpy.getCall(4).args[0]).toEqual({
+        key: 'scenario',
+        value: inputValue.scenario,
+      });
+      expect(validateSequenceSpy.called).toEqual(true);
+    });
+
+    it('should call update sequence a single time followed by validate sequence if action is clear', () => {
```
Comment (kkoskelin) [inline web-client/src/ustc-ui/utils/documentTypeSelectHelper.test.js]: perhaps `if "action" is "clear"` or some other formatting to make the test description clearer to a casual reader

**Review by kkoskelin — APPROVED** [review]

**Review by ericsorenson — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**